package.json

```diff
@@ -20,7 +20,8 @@
     "ace-builds": "1.4.12",
     "axios": "1.6.0",
     "core-js": "3.25.1",
-    "date-fns": "2.28.0",
+    "date-fns": "3.0.0",
```

Comment:
any reason we don't want to update to the latest, 3.6.0?

Reply:
Oh good call. All the marketing info on their website is still celebrating the 3.0.0 release so I was fooled into thinking that's the latest version.